pyccel/codegen/printing/fcode.py

```diff
@@ -616,8 +616,6 @@ def _print_PythonSum(self, expr):
     def _print_PythonReal(self, expr):
         value = self._print(expr.internal_var)
         return 'real({0})'.format(value)
-    def _print_PythonFloat(self, expr):
-        return expr.fprint(self._print)
```

Comment:
We had two methods with the same name?? 😱

Reply:
By the way, I've just noticed that we have a `_print_Real` method... Do you know what it's used for?

Reply:
Yes, apparently!

Hmm, you are right. I suspect it used to print `NumpyReal` (which prints the real part of a complex number) before it was renamed. Codacy confirms that it is unused so I have deleted it. See 7177ad1

Reply:
Thanks!